release-notes/10.0/preview/preview1/winforms.md

```diff
@@ -1,14 +1,69 @@
+- `public virtual object? GetData(Type format)`
+
+**ClipboardProxy (VisualBasic wrapper for WinForms Clipboard):**
+
+- `public object GetData(string format)`
+
+### New APIs
+
+We’ve introduced the following APIs in Windows Forms for .NET 10 Preview 1:
 
-- [What's new in Windows Forms in .NET 10](https://learn.microsoft.com/dotnet/desktop/winforms/whats-new/net100) documentation
+**Clipboard:**
 
+- `public static void SetDataAsJson<T>(string format, T data)`
+- `public static bool TryGetData<T>(string format, out T data)`
+- `public static bool TryGetData<T>(string format, Func<Reflection.Metadata.TypeName, Type> resolver, out T data)`
+
+**IDataObject:**
+**ITypedDataObject:**
+
+- `public void SetDataAsJson<T>(T data)`
+- `public void SetDataAsJson<T>(string format, T data)`
+- `public void SetDataAsJson<T>(string format, bool autoConvert, T data)`
+- `public bool TryGetData<T>(out T data);`
+- `public bool TryGetData<T>(string format, out T data);`
+- `public bool TryGetData<T>(string format, bool autoConvert, out T data);`
+- `public bool TryGetData<T>(string format, Func<Reflection.Metadata.TypeName, Type> resolver, bool autoConvert, out T data);`
+- `protected virtual bool TryGetDataCore<T>(string format, Func<Reflection.Metadata.TypeName, Type> resolver, bool autoConvert, out T data)`
+
+**DataObjectExtensions:**
+
+- `public static bool TryGetData<T>(this IDataObject dataObject, out T data)`
+- `public static bool TryGetData<T>(this IDataObject dataObject, string format, out T data)`
+- `public static bool TryGetData<T>(this IDataObject dataObject, string format, bool autoConvert, out T data)`
+- `public static bool TryGetData<T>(this IDataObject dataObject, string format, Func<Reflection.Metadata.TypeName, Type> resolver, bool autoConvert, out T data)`
+
+**ClipboardProxy:**
+
+- `public void SetDataAsJson<T>(T data)`
+- `public void SetDataAsJson<T>(string format, T data)`
+- `public bool TryGetData<T>(string format, out T data)`
+- `public bool TryGetData<T>(string format, System.Func<System.Reflection.Metadata.TypeName, System.Type> resolver, out T data)`
+
+### New Configuration Switch
+
+If an application is unable to adopt the new APIs, and requires a fallback to `BinaryFormatter` serialization during Clipboard and Drag/Drop operations, the developer will need to take additional action. In .NET 9, calls to BinaryFormatter will always throw a `PlatformNotSupportedException` unless the developer follows the instructions located here: [BinaryFormatter compatibility package](https://learn.microsoft.com/dotnet/standard/serialization/binaryformatter-migration-guide/compatibility-package) . In .NET 10 Preview 1, WinForms will now also by default disable the use of BinaryFormatter as a fallback unless it is explicitly enabled with this new configuration switch. This switch is `Windows.ClipboardDragDrop.EnableUnsafeBinaryFormatterSerialization`, and it must be set to `True` for the fallback to happen successfully.

```

Comment:
typo: extra white space after the closing paren.

Reply:
`In .NET 10 Preview 1, WinForms will now also by default disable`  ->> `In .NET 10 Preview 1, WinForms will also by default disable`

Reply:
Done, thanks!